src/main/java/coordinate/Point.java

```diff
@@ -0,0 +1,69 @@
+package coordinate;
+
+import java.util.Objects;
+
+public class Point {
+    private final int x;
+    private final int y;
+
+    private Point(int x, int y) {
+        this.x = x;
+        if (x < 0 || x > 24) {
+            throw new IllegalArgumentException();
+        }
```

Comment:
유효성 검사를 실패하게 되면 인자를 주입할 필요가 없기 때문에 유효성 검사를 진행한 후 x에 값이 주입되어야 합니다. 
```suggestion
       if (x < 0 || x > 24) {
            throw new IllegalArgumentException();
        }
        
        this.x = x;
```